package.json

```diff
@@ -62,10 +62,12 @@
   "main": "built/index.js",
   "typings": "built/index.d.ts",
   "scripts": {
+    "postinstall": "cd bin && webdriver-manager update && npm run install_website && npm run install_testapp",
```

Comment:
I am also not a fan of this step. We should avoid `postinstall` because not everyone is a contributor.